src/components/CompareProject.vue

```diff
@@ -29,58 +87,35 @@ export default defineComponent({
       default: false,
     },
     options: {
-      type: Array,
+      type: Array as PropType<Option[]>,
       default() {
-        return [
-          {
-            description: "I don't see any change between the two images.",
-            iconColor: '',
-            title: 'No change',
-            mdiIcon: 'mdi-equal',
-            shortkey: 1,
-            value: 0,
-          },
-          {
-            description: 'There is change between the two images.',
-            iconColor: 'green',
-            title: 'Change',
-            mdiIcon: 'mdi-not-equal-variant',
-            shortkey: 2,
-            value: 1,
-          },
-          {
-            description: 'I am not sure.',
-            iconColor: 'orange',
-            title: 'Not sure',
-            mdiIcon: 'mdi-head-question',
-            shortkey: 3,
-            value: 2,
-          },
-          {
-            description: 'The imagery is bad or clouded.',
-            iconColor: 'red',
-            title: 'Bad imagery',
-            mdiIcon: 'mdi-weather-cloudy',
-            shortkey: 4,
-            value: 3,
-          },
-        ]
+        return defaultOptions
       },
     },
     project: {
-      type: Object,
+      type: Object as PropType<ProjectCompareType>,
       required: true,
     },
     tasks: {
-      type: Array,
+      type: Array as PropType<Task[]>,
       required: true,
     },
     tutorial: {
-      type: Object,
-      require: false,
+      type: Object as PropType<Tutorial>,
+      required: false,
     },
   },
-  data() {
+  data(): {
+    allAnswered: boolean
+    arrowKeys: boolean
+    overlay: boolean
+    results: Record<string, number | undefined>
+    startTime: string | null
+    task: object
+    taskId: string | undefined
+    taskIndex: 0
```

Comment:
Is this datatype correct? @frozenhelium shouldn't this be `number`?

Reply:
for `taskIndex`, yes!